app/modules/usb/views/usb_listing.php

```diff
@@ -25,6 +25,7 @@
 			<th data-i18n="usb.device_speed" data-colname='usb.device_speed'></th>
 			<th data-i18n="usb.internal" data-colname='usb.internal'></th>
 			<th data-i18n="usb.media" data-colname='usb.media'></th>
+			<th data-i18n="usb.product_id" data-colname='usb.product_id'></th>
```

Comment:
Don't forget to increase the `listing.loading` on line 34 when you add a column to the listing. The count should be `10`